src/protocol/codecs/tagmsg.rs

```diff
@@ -30,6 +33,31 @@ pub enum Tag {
     VoteBundle,
 }
 
+impl Display for Tag {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "{}",
+            match self {
```

Comment:
I'd suggest using the explicit names so a look at this match statement isn't necessary to understand logs; unless there's a reason for these shorthands?

Reply:
> Maybe I could have implemented a normal (non Display) function to get these strings?

From our internal offline discussion: **Yes!**

The `Display` trait should be [used for user-facing output](https://doc.rust-lang.org/std/fmt/trait.Display.html):

> Display is similar to [Debug](https://doc.rust-lang.org/std/fmt/trait.Debug.html), but Display is for user-facing output, and so cannot be derived.